swift/apple/FirezoneKit/Sources/FirezoneKit/NetworkSettings.swift

```diff
@@ -96,7 +89,7 @@ public struct NetworkSettings: Equatable {
   // MARK: - NEPacketTunnelNetworkSettings Builder
 
   /// Build NEPacketTunnelNetworkSettings from current state
-  private func buildNetworkSettings() -> NEPacketTunnelNetworkSettings? {
```

Comment:
Having to make this `public` is a bummer because it means our unit-tests can no longer enforce that we only create `NEPacketTunnelNetworkSettings` when something differs because now, this can be called whenever and we don't enforce that you went through the `update` functions first.